src/osemgrep/cli_interactive/Framed_zipper.mli

```diff
@@ -43,6 +43,7 @@ type 'a t
 val move_left : 'a t -> 'a t
```

Comment:
maybe rename move_up, move_down? so it's close to the scrolling metaphor? (even though there are horizontal scrolling it's true; still in the use case of semgrep interactive it's up/down scrolling).